content/en/tests/setup/dotnet.md

```diff
@@ -361,6 +367,34 @@ await module.CloseAsync();
 
 Always call `module.Close()` or `module.CloseAsync()` at the end so that all the test data is flushed to Datadog.
 
+## Best Practices
+
+### Test session name `DD_TEST_SESSION_NAME`
+
+Use `DD_TEST_SESSION_NAME` to define the test session name for your tests (`test_session.name` tag). Use this to identify a group of tests. Examples of values for this tag would be:
```

Comment:
I find this a little confusing. Does `DD_TEST_SESSION_NAME` both define the test session name and also identify a group of tests?

Reply:
it's the same. The problem is that the concept "test session" is mostly datadog's, so it needs some clarifying. Can you think of a better way to express this?